src/Umbraco.Web.UI/Umbraco/Views/Preview/Index.cshtml

```diff
@@ -1,14 +1,13 @@
-@using Umbraco.Core
-@using ClientDependency.Core
-@using ClientDependency.Core.Mvc
-@using Umbraco.Web.Composing
-@using Umbraco.Web
+﻿@using Umbraco.Web.Composing
 
 @inherits System.Web.Mvc.WebViewPage<Umbraco.Web.Editors.BackOfficePreviewModel>
 @{
     var disableDevicePreview = Model.DisableDevicePreview.ToString().ToLowerInvariant();
 
-    Html.RequiresCss("assets/css/canvasdesigner.css", "Umbraco");
+    var runtimeMinifier = Current.RuntimeMinifier;
+
+    runtimeMinifier.RequiresCss("umbraco-preview",
```

Comment:
This needs to be a pre-defined bundle... as in, defined in c# on startup. View based declarations will not work in load balancing (see prev comments). I'll update.